oop3-abstract-morph.js

```diff
@@ -9,22 +9,97 @@ Create at least 3 sub-classes that inherit from the Abstract Creature Class.  Ex
     -Reptile
     -Fish
 
-These sub-classes MUST implement the abstract methods of move() and act(), which are found on the abstract Creature class. In addition to the 2 abstract methods, implement 1 unique method on each sub-class.  Also, each sub-class MUST have at least 2 unique property fields on them.  Example property fields are:
+These sub-classes MUST implement the abstract methods of move() and act(), which are found on the abstract Creature class. In addition to the 2 abstract methods, implement 1 unique method on each sub-class.  
+//Also, each sub-class MUST have at least 2 unique property fields on them.  Example property fields are:
     -name
     -weight
     -food
     -age
 
 */
 
-    class Creature {
-        //your code here...
+class Creature {
+    constructor() {
+        if (this.constructor == Creature) {
+            throw new Error("Unable to instantiate Abstract Class creature")
+        }
     }
+    move() {
+        throw new Error("This is an abstract method.")
+    };
+    act() {
+        throw new Error("This is an abstract method.")
+    };
+}
+
+class Human extends Creature {
+    //your code here...
+    #name;
```

Comment:
if all of the objects that extend a super class have the same method or field, you should include it in the definition of the super class. Less to repeat over and over every time.